Makefile

```diff
@@ -24,14 +19,16 @@ bin/test: $(TEST_OBJ)
 	mkdir -p $(@D)
 	$(CXX) -o $@ $^ $(LDFLAGS) -lstdc++ -lm
 
-cuda_build_test: $(CUDA_TEST_SRC) $(CUDA_DEPS)
-	$(CXX) -I. -c $< $(CFLAGS) $(CXXFLAGS) $(CUDA_FLAGS)
-	
+cuda_build_test: $(CUDA_TEST_SRC) $(DEPS)
+	$(CXX) -I. -c $< $(CFLAGS) $(CXXFLAGS) --cuda-gpu-arch=sm_52 -nocudalib -nocudainc -emit-llvm
+	# TODO: Figure out how to run this build test to produce outputs in bin/ or obj/
+	rm *.bc
```

Comment:
Yeah, I couldn't get it to work. -o doesn't work because -emit-llvm "produces multiple outputs".

Reply:
Same, and it was surprisingly stubborn to work around. I gave up.